src/main.cc

```diff
@@ -389,10 +399,24 @@ int main(int argc, char **argv) {
     return 0;
   }
 #endif /* Linux || FreeBSD || Haiku || NetBSD || OpenBSD */
+  ////////
+  ///
+  ///
+  ///
+  ///
+  ///
+  ///
+  /// ///
+  /// ///
+  ///
+  ///
+  ///
+  ///
+  /// ////
 
   try {
     set_current_config();
-
+    tryToReplaceAliasToPresetPath(THEME_PRESETS_REPO_PATH, THEME_PRESETS_REPO_CLONING_URL );
```

Comment:
`set_current_config` resolves paths to configs and _then_ you replace theme names with paths?

Shouldn't `tryToReplaceAliasToPresetPath` logic go into `set_current_config`, and before we do path resolution?

Ideally, there should be some easily recognizable disambiguation between a system path and a theme/preset name. Something like `theme:herpy_derpy_der`, so that you can easily and cheaply assume it's a theme, and not a directory relative to cwd (without asking the FS).